### PR TITLE
fix: Prevent opening thumbnail by reaction click

### PR DESCRIPTION
**Pull request by HoonBaek**
### Description Of Changes

* Prevent opening the FileViewer by the reaction onClick event

[UIKIT-3977](https://sendbird.atlassian.net/browse/UIKIT-3977)

[UIKIT-3977]: https://sendbird.atlassian.net/browse/UIKIT-3977?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/ui/ReactionButton/index.tsx [hunk 1]
```diff
@@ -18,7 +21,7 @@ export interface ReactionButtonProps {
     e: MouseEvent<HTMLDivElement> | KeyboardEvent<HTMLDivElement> | TouchEvent<HTMLDivElement>
   ) => void;
 }
-const ReactionButton = React.forwardRef((props: ReactionButtonProps, ref: RefObject<HTMLDivElement>): ReactElement => {
+const ReactionButton: ForwardRefExoticComponent<ReactionButtonProps> = React.forwardRef((props: ReactionButtonProps, ref: ForwardedRef<HTMLDivElement>) => {
```
Comment (sravan-s) [inline src/ui/ReactionButton/index.tsx:24]: ForwardRefExoticComponent 😮

Comment (sravan-s) [reply src/ui/ReactionButton/index.tsx:24]: nit: the line is too long.. I thought we set 120 as limit in eslint

**Review by sravan-s — APPROVED** [review]

File: src/ui/ReactionButton/index.tsx (hunk 1 above)
Comment (AhyoungRyu) [reply src/ui/ReactionButton/index.tsx]: We can change the max line num too ;)

File: src/ui/ReactionButton/index.tsx [hunk 2]
```diff
@@ -28,6 +31,14 @@ const ReactionButton = React.forwardRef((props: ReactionButtonProps, ref: RefObj
     children,
   } = props;
 
+  const onClickHandler = useLongPress({
+    onLongPress: noop,
+    onClick: onClick,
```
Comment (AhyoungRyu) [inline src/ui/ReactionButton/index.tsx:36]: (Don't need to apply in this PR. I'm just commenting as a reminder for myself)

```suggestion
    onClick,
```

Could be same too. Let me turn on object-shorthand eslint rule later :) 
https://github.com/sendbird/sendbird-uikit-react/blob/main/.eslintrc.json#L81

**Pull request merged**